### ISSUE TITLE
Motorola Droid Turbo 2 Phonegap Pdf417 plugin issue

### ISSUE THREAD

**Issue by sirius2013** [PDF417/pdf417-phonegap#18]
![image](https://cloud.githubusercontent.com/assets/5020713/11523777/2f4fae70-9900-11e5-8434-a5f3d178ab6e.png)

When run pdf417 plugin, I got above error.
current phonegap plugin support Motorola device?

**Comment by sirius2013** [comment]
This is details' spec for my droid.
![image](https://cloud.githubusercontent.com/assets/5020713/11523809/5cdfd306-9900-11e5-95db-326c13d2a1f1.png)

**Comment by DoDoENT** [comment]
Hi,

without the ADB logs I can only guess what went wrong. Does this happen with our demo app or only after you have integrated PDF417.mobi plugin into your app?

We tried to reproduce the error on Nexus 5X (which should have similar hardware to Turbo 2) and could not reproduce it.

If the error appears in our demo app, can you provide us with ADB log so we could analyse the problem? If the error appears only after integrating in your app, then it may be possible that you are also integrating other plugins that have improper support for 64-bit ARM processors. You can see more information about this case in our [native android SDK documentation](https://github.com/PDF417/pdf417-android#combineNativeLibraries).

**Comment by sirius2013** [comment]
Hi, Thank you for your message.

<string name="photopayErrorInitializing">Failed to initialize native library.</string> 
It says it is related with "photopayErrorInitializing", but not sure what is this.

**Comment by DoDoENT** [comment]
Does this happen with our demo app or only with your app?

Can you attach a stack trace here?

**Comment by sirius2013** [comment]
I tried to build demo app, but failed, and opened new issue.
https://github.com/PDF417/pdf417-phonegap/issues/19

**Comment by DoDoENT** [comment]
Hi @sirius2013,

is this issue resolved for you? Can we close it?

**Comment by sirius2013** [comment]
ok
